### PR TITLE
Conclusion

### LINKED ISSUE #5: Conclusion

### PR DESCRIPTION
**Pull request by steffansluis**
The conclusion to the research questions. Closes #5 and #27.

### CODE REVIEW

**Comment by WouterKvG** [comment]
@steffansluis @bartist Rebase is gedaan. Alleen rq3 moet nog en rq2 moet gereviewed worden.

File: conclusions/rq2.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+- modular code with standartized documentation, easy set-up and tutorials;
+- use both peer reviews as formal testing;
+- further stimulation of intrinsic and extrinsic motivation.
+
+A small core team is necessary to provide a stable basis on which the rest of the community can rely on. This team should contain members respected by the community and provide clear information about the state and future of the project. In relation to FeedbackFruits, this team can, consist of one person from FeedbackFruits and two or three persons from the community. The members from the community can change per project and at least project issuer should take a place in the team.
+
+GitHub is a proven tool that is familiar to almost every software developer. Studies have shown that is a very effective way of communication between the community. However, when the code is difficult to understand, not properly tested or document. Or when it is difficult to set-up the barrier for a new user to contribute is much higher. The use of modern tools and new techniques enables people to learn and intrinsically motivates people to contribute.
```
Comment (bartist) [inline conclusions/rq2.md]: "document. Or " --> "document, or"

File: conclusions/rq1.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+## What can be learned from existing (non-software) communities?
+Looking at the different communities, it can be concluded that most of the communities have one thing in common: they work because they have a goal. In the case of the intentional communities: all but the practical communities have their ideals as a goal to fulfill. The practical communities lack a goal, but those communities work because they fulfill the goal of making life easier.
```
Comment (steffansluis) [inline conclusions/rq1.md]: `The practical communities lack a goal, but those communities work because they fulfill the goal of making life easier.`

I would rephrase this to be something like: `The practical communities make life easier rather than fulfilling an explicit goal.`

File: conclusions/rq1.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+## What can be learned from existing (non-software) communities?
+Looking at the different communities, it can be concluded that most of the communities have one thing in common: they work because they have a goal. In the case of the intentional communities: all but the practical communities have their ideals as a goal to fulfill. The practical communities lack a goal, but those communities work because they fulfill the goal of making life easier.  
+The goal of MMO-RGP communities are to make some part of the game-play possible or easier. It is because some quests can only be done in a party that forming one is a goal to achieve. Late in the game, when enemies can become exponentialy stronger, the necessity of trust-worthy party members becomes greater. This can be the reason and the goal to form a guild. A clear and good goal can make a guild stable and well-defined.
```
Comment (steffansluis) [inline conclusions/rq1.md]: The goal of MMO-RGP communities **is**

File: conclusions/rq3.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# How are requirements established and used in software development?
+
+Section 2 describes the process of requirements engineering. Implementing structured requirements development and management early on can help deal with the challenges of building a large software systems. One of the main challenges is coordinating dependencies between requirements, and can be dealt with using coordination mechanisms.
```
Comment (WouterKvG) [inline conclusions/rq3.md]: `is coordinating dependencies between requirements, and can be dealt with ..` Seems like an odd sentence. Maybe split the sentence like `between requirements. Coordination mechanisms can solve these challenges.`

File: conclusions/rq1.md (hunk 3 above)
Comment (steffansluis) [inline conclusions/rq1.md]: `It is because` ==> `This is because`

Comment (steffansluis) [inline conclusions/rq1.md]: Late**r** in the game

Comment (steffansluis) [inline conclusions/rq1.md]: `the necessity of trust-worthy party members becomes greater`

I think necessity becomes higher and need becomes greater. Not sure though...

File: conclusions/rq3.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+
+Additionally, the requirements engineering process should be kept iterative. The Twin Peaks model facilitates this and can be extended to work in conjunction with software architecture and design processes. This makes it possible to develop software modularly while still being able to verify the software meets its requirements.
```
Comment (WouterKvG) [inline conclusions/rq3.md]: Change to `software meets its requirements` ==> `software meets the set requirements`?

File: conclusions/rq1.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
...
+Looking at the different communities, it can be concluded that most of the communities have one thing in common: they work because they have a goal. In the case of the intentional communities: all but the practical communities have their ideals as a goal to fulfill. The practical communities lack a goal, but those communities work because they fulfill the goal of making life easier.  
+The goal of MMO-RGP communities are to make some part of the game-play possible or easier. It is because some quests can only be done in a party that forming one is a goal to achieve. Late in the game, when enemies can become exponentialy stronger, the necessity of trust-worthy party members becomes greater. This can be the reason and the goal to form a guild. A clear and good goal can make a guild stable and well-defined.  
+Software communities often have the goal of improving the quality of the software product. While the users report bugs or elements of the program that they're unsatisfied about, the developers can improve the program using the feedback from the users. This will result in greater users-satisfaction, wich is profitable for both user and developer.
```
Comment (steffansluis) [inline conclusions/rq1.md]: `they're` ==> `they are`

No contractions in formal text

File: conclusions/rq1.md [hunk 7]
```diff
@@ -0,0 +1,6 @@
...
+
+Every community that untakes something must have a clear and relatable goal. This goal is the foundation for the stability of the community and ensures that a community is worth joining.
```
Comment (steffansluis) [inline conclusions/rq1.md]: `that untakes`

undertakes?

File: conclusions/rq2.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
+## What can be learned from existing efforts to involve a community with software development?
+In short, there are a few key factors that can be identified in succesful software development communities:
+- a small core team that actively communicates about the current and desired state of the software;
+- use of GitHub (git, pull-requests, issue-tracker) or a similar system;
+- modular code with standartized documentation, easy set-up and tutorials;
```
Comment (steffansluis) [inline conclusions/rq2.md]: standar**d**ized

File: conclusions/rq2.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+- use both peer reviews as formal testing;
```
Comment (steffansluis) [inline conclusions/rq2.md]: `use both peer reviews as formal testing` ==> `use  of both peer reviews as well as formal testing`

File: conclusions/rq2.md [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+- further stimulation of intrinsic and extrinsic motivation.
+
+A small core team is necessary to provide a stable basis on which the rest of the community can rely on. This team should contain members respected by the community and provide clear information about the state and future of the project. In relation to FeedbackFruits, this team can, consist of one person from FeedbackFruits and two or three persons from the community. The members from the community can change per project and at least project issuer should take a place in the team.
```
Comment (steffansluis) [inline conclusions/rq2.md]: `on which the rest of the community can rely on.`

Either `on which the rest of the community can rely.` or `which the rest of the community can rely on.`

Comment (steffansluis) [inline conclusions/rq2.md]: `In relation to FeedbackFruits, this team can, consist of one`

Why the second comma? The first can be left out as well

Comment (steffansluis) [inline conclusions/rq2.md]: `two or three persons`

The plural of person is people :) (except when talking about companies in terms of law and stuff)

File: conclusions/rq2.md (hunk 1 above)
Comment (steffansluis) [inline conclusions/rq2.md]: `way of communication between the community` ==> way of communicati**ng within** the community

Comment (steffansluis) [inline conclusions/rq2.md]: not properly tested or document**ed**

Also, this sentence shouldn't end here, but have a comma instead of a period

Comment (steffansluis) [inline conclusions/rq2.md]: `difficult to set-up`

I think `set up` should actually not have a dash here. Also, I think a comma would be nice after `set up`
